cbmc_viewer/doc/make-clause.md

```diff
@@ -0,0 +1,33 @@
+# Name
+
+make-clause -- list solver query complexity metrics
+
+# Synopsis
+
+	usage: make-clause [-h] [--clause FILE] [--srcdir SRCDIR] 
+                        --dimacs DIMACS --core CORE [--verbose]
+                       [--debug]
+
+# Description
+
+This is a front end for the clauset module that viewer uses to scan
+the results of cbmc solver query complexity metric collection. The
+output is a json file that describes a mapping between clauses and 
+lines of program code. Additionally, it highlights instructions that
+contribute to the UNSAT core and also program locations that do not 
+contribute any clauses to the UNSAT core.
+
+Simple uses of make-clause are
+
+    # Generate the list of solver query complexity metric from output of 
+    # "cbmc --write-solver-stats-to"
+    # cbmc --write-solver-stats-to clause.json program.goto
+    # Generate dimacs cnf formula 
+    # cbmc program.goto --dimacs --outfile dimacs.cnf
+    # Extract UNSAT core
+    # kissat dimacs.cnf proof
+    # drat-trim dimacs.cnf proof -c core
```

Comment:
Where does this long sequence of preprocessing happen?  Do you assume it is happening in the Makefile?

Reply:
Its done in the Makefile.

Reply:
Let's add a comment that this is done in the makefile before calling this program.